### DISCUSSION TITLE
zkSyinc pulgging  test net deploying error!

### DISCUSSION THREAD

**Discussion by Isac998** [Cyfrin/foundry-full-course-cu#2325] [General]
![image](https://github.com/user-attachments/assets/09028e9b-6283-4317-9162-5596f24a5d9d)

![image](https://github.com/user-attachments/assets/74020c9c-2ce7-41be-9945-25c67f756259)

![image](https://github.com/user-attachments/assets/bc7a1414-d071-42b3-ae7a-05bab2457403)

**Comment by EngrPips** [comment]
Hello @Isac998, working with this `zksync` plug-in on remix can be kind of difficult, you just need to retry some other time and it should work. Sometimes the plugin works appropriately, and sometimes it doesn't. Don't let that slow you down, as Patrick always stresses that `zksync` stuff is optional for the course.

**Reply by Isac998** [reply]
@EngrPips  alright, thanks a ton man ! Imma move on and retry some other time!

**Reply by EngrPips** [reply]
You are welcome, Friend.
